test/libsolidity/syntaxTests/immutable/writing_after_initialization_modifier.sol

```diff
@@ -0,0 +1,12 @@
+contract C {
+    uint immutable x = 0;
+
+    function f() readX internal { }
+
+    modifier readX() {
+        _; x = 1;
+    }
+}
+// ----
+// TypeError: (111-112): Immutable variables can only be initialized directly in the constructor.
```

Comment:
Did we already decide on whether initializing at point of declaration is possible or not?

Reply:
Ah sorry, confused the error.

Reply:
ah, I actually thought it is okay (in which case the error message is wrong and needs to be changed)